### ISSUE TITLE
Doesn't work anymore

### ISSUE THREAD

**Issue by kilinkis** [fdaciuk/react-trianglify#13]
Sadly, this wrapper just doesn't work anymore.
I tried it with gatsby and create-react-app, with and without typescript.

in all cases I see this error:

```
Failed to compile
./node_modules/react-trianglify/src/index.js
SyntaxError: /Users/kilinkis/Documents/dev/my-app/node_modules/react-trianglify/src/index.js: Unexpected token (39:10)

  37 |         method: 'generateCanvas',
  38 |         component: ({ canvasRef, height, width }) => (
> 39 |           <canvas ref={canvasRef} height={height} width={width} />
     |           ^
  40 |         )
  41 |       },
```
it happens for every return type.

and then, even if I comment those out, then I see this error

```
Failed to compile
./node_modules/react-trianglify/src/index.js
Module not found: Can't resolve 'trianglify' in '/Users/kilinkis/Documents/dev/my-app/node_modules/react-trianglify/src'
```

**Comment by fdaciuk** [comment]
Hey @kilinkis! Did you install `trianglify` in your project? Trianglify is a peer dependency for this library :)

**Comment by fdaciuk** [comment]
The first error is fixed on `2.0.5`, but you still must install `trianglify` and `react-trianglify` :)

**Comment by kilinkis** [comment]
hi @fdaciuk, thank you so much for your reply.
Indeed, I have not installed `trianglify`, my bad, it works like a charm now.
Thank you for fixing the other one!
